### PR TITLE
Sample data

### PR DESCRIPTION
**Pull request by hagaram**
Sample data for Booking Manager Added (no tests so far)

### CODE REVIEW

File: BookingManager-sampleData/src/main/java/cz/muni/fi/pa165/brown/SampleDataLoadingFacadeImpl.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        room4 = room(4, new BigDecimal(20), 4, hotel2);
+        room5 = room(5, new BigDecimal(70), 2, hotel2);
+
+
+
+        reservation1 = reservation(date1, date2, user1, room1);
+        reservation2 = reservation(date1, date2, user2, room2);
+        reservation3 = reservation(date4, date5, user2, room2);
+        reservation4 = reservation(date3, date4, user3, room4);
+
```
Comment (vasekstebra) [inline BookingManager-sampleData/src/main/java/cz/muni/fi/pa165/brown/SampleDataLoadingFacadeImpl.java:86]: You should save it all to the database through the appropriate services

**Review by vasekstebra — CHANGES_REQUESTED** [review]

File: BookingManager-sampleData/src/main/java/cz/muni/fi/pa165/brown/SampleDataLoadingFacadeImpl.java (hunk 1 above)
Comment (Dominick1993) [reply BookingManager-sampleData/src/main/java/cz/muni/fi/pa165/brown/SampleDataLoadingFacadeImpl.java:86]: He does exactly that in the methods that are called, e.g. in `room(...)` method a `roomService.create(room);` is called.

**Review by Dominick1993 — APPROVED** [review]

File: BookingManager-sampleData/src/main/java/cz/muni/fi/pa165/brown/SampleDataLoadingFacadeImpl.java (hunk 1 above)
Comment (vasekstebra) [reply BookingManager-sampleData/src/main/java/cz/muni/fi/pa165/brown/SampleDataLoadingFacadeImpl.java:86]: Dorty. My mistake - I  overlooked it.

Comment (vasekstebra) [reply BookingManager-sampleData/src/main/java/cz/muni/fi/pa165/brown/SampleDataLoadingFacadeImpl.java:86]: * Sorry - bad autocorrect on phone. :-)

**Pull request merged**